src/utils/mongoose/index.ts

```diff
@@ -98,11 +98,12 @@ export class MongooseDatabaseInterface extends DatabaseInterface {
             if (result.nUpserted > 0) {
                 logInfo({}, `Added ${result.nUpserted} ABIs`)
             }
-            return result
+            return true
         } catch (e) {
             console.log('abi mongoose error')
```

Comment:
Preferably we catch the specific error they throw (`Unauthorized`) and swallow it silently, and any other error, we can log an error for

Reply:
I just updated this but in hindsight it was not a priority and I should not have spent time on it